sql/core/src/main/scala/org/apache/spark/sql/execution/GeneratedAggregate.scala

```diff
@@ -41,13 +42,15 @@ case class AggregateEvaluation(
  * @param groupingExpressions expressions that are evaluated to determine grouping.
  * @param aggregateExpressions expressions that are computed for each group.
  * @param child the input data source.
+ * @param unsafeEnabled whether to allow Unsafe-based aggregation buffers to be used.
  */
 @DeveloperApi
 case class GeneratedAggregate(
```

Comment:
We should consider breaking this into two operators at some point so the explain is more readable.